builtin/providers/google/resource_google_project.go

```diff
@@ -172,6 +178,21 @@ func resourceGoogleProjectCreate(d *schema.ResourceData, meta interface{}) error
 		}
 	}
 
+	// Set the billing account
+	if v, ok := d.GetOk("billing_account"); ok {
+		ba := cloudbilling.ProjectBillingInfo{
+			BillingAccountName: "billingAccounts/" + v.(string),
+		}
+		_, err = config.clientBilling.Projects.UpdateBillingInfo("projects/"+pid, &ba).Do()
+		if err != nil {
+			d.Set("billing_account", "")
+			if _err, ok := err.(*googleapi.Error); ok {
```

Comment:
Is there a difference between how `*googleapi.Error` and `error` get printed when using `%v`?